src/server.c

```diff
@@ -3114,7 +3114,14 @@ void homekit_setup_mdns(homekit_server_t *server) {
         return;
     }
 
-    homekit_mdns_configure_init(name->value.string_value, PORT);
+    char unique_name[65]={0};
```

Comment:
Maybe it make sense to make buffer size dynamic?

Reply:
Regarding name[64]: it will be truncated at 58, (just as you truncate at 64 in homekit_mdns_configure_init). No great loss IMO since a 64 char name is unreadable already.

@RavenSystem: it would be better to use the MAC, I agree.  
However, using MAC, I think it is not so easily picked up in server.c without including <esp-wifi.h>
So, is your opinion to include eps-wifi.h or to add an attribute in the accessory_info to convey the MAC?? I know Max, you like your code to be clean and modular so that is why I do not assume to make shortcuts ;-)

Reply:
Ok, makes sense.